arex-storage-web-api/src/main/java/com/arextest/storage/beans/IndexesSettingConfiguration.java

```diff
@@ -43,20 +41,17 @@ private void ensureMockerQueryIndex(MongoDatabase database) {
                 AREXMocker.class);
 
         try {
-          Document index = new Document();
-          index.append(AREXMocker.Fields.recordId, 1);
-          collection.createIndex(index);
+          collection.dropIndexes();
```

Comment:
What is the purpose of this method？
Delete all indexes first and then recreate them?

Reply:
<img width="715" alt="image" src="https://github.com/arextest/arex-storage/assets/35320326/b1f6f505-15eb-4e04-aca6-d4de262fc7b1">
There have been operations to delete specific indexes before, which cause lots of error logs. Now it has been changed to delete all indexes.